### PR TITLE
Inject rocm runtimes to runtime-images folder

### PR DESCRIPTION
**Pull request by atheo89**
Related to: https://issues.redhat.com/browse/RHOAIENG-9680
Depends on: https://github.com/openshift/release/pull/54567

## Description
Add rocm runtimes to runtime-images folder

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-pytorch-ubi9-py39.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "display_name": "Pytorch with ROCm and Python 3.9 (UBI9)",
+    "metadata": {
+      "tags": [
+        "rocm-pytorch"
+      ],
+      "display_name": "Pytorch with ROCm and Python 3.9 (UBI9)",
+      "image_name": "NEEDS TO BE UPDATED",
```
Comment (jiridanek) [inline jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-pytorch-ubi9-py39.json:8]: todo

File: jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-tensorflow-ubi9-py39.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "display_name": "TensorFlow with ROCm and Python 3.9 (UBI9)",
+    "metadata": {
+      "tags": [
+        "rocm-tensorflow"
+      ],
+      "display_name": "TensorFlow with ROCm and Python 3.9 (UBI9)",
+      "image_name": "NEEDS TO BE UPDATED",
```
Comment (jiridanek) [inline jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-tensorflow-ubi9-py39.json:8]: todo

**Review by jiridanek — COMMENTED** [review]
looks like the other json files around, so I guess it should be correct

File: jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-pytorch-ubi9-py39.json (hunk 1 above)
Comment (atheo89) [reply jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-pytorch-ubi9-py39.json]: done

File: jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-tensorflow-ubi9-py39.json (hunk 2 above)
Comment (atheo89) [reply jupyter/datascience/ubi9-python-3.9/runtime-images/rocm-tensorflow-ubi9-py39.json]: done

**Comment by atheo89** [comment]
This PR is ready for review

**Comment by jiridanek** [comment]
/lgtm

* https://quay.io/repository/opendatahub/workbench-images/manifest/sha256:519b7f4ea9a16065ba2213389de4e38ee4afa3084f3b8b770157818928ad0f7a
* https://quay.io/repository/opendatahub/workbench-images/manifest/sha256:a1bbf6153921b60cb58f094d110bf6dc054d13a07c4c513a6b994d1b74a8b60e?tab=layers

the images as displayed on quay.io look to be the correct ones

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

thanks for the work :100:

**Pull request merged**